pkg/transport/cmd/options.go

```diff
@@ -51,7 +55,8 @@ func (options *TransportOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&options.Concurrency, "concurrency", options.Concurrency, "number of concurrent workers to run in parallel")
 	options.WdsClientOptions.AddFlags(fs)
 	options.TransportClientOptions.AddFlags(fs)
-	fs.IntVar(&options.MaxSizeWrappedObject, "max-size-wrapped-object", options.MaxSizeWrappedObject, "Max size of the wrapped object")
+	fs.IntVar(&options.MaxSizeWrapped, "max-size-wrapped", options.MaxSizeWrapped, "Max size of the wrapped object in bytes")
```

Comment:
Is "size of the wrapped object" accurate? I thought that the OCM limit is on the size of the slices of manifests in the spec, which is a few steps away from the size of the ManifestWork.